src/shim.c

```diff
@@ -939,6 +940,15 @@ connect_to_proxy(struct cc_shim *shim)
 	return false;
 }
 
+/*
+ * Print version information.
+ */
+void
+show_version(void) {
+	printf("%s version: %s\n", PACKAGE_NAME, PACKAGE_VERSION);
+	printf("commit: %s\n", GIT_COMMIT);
```

Comment:
@amshinde - I think it would be best if you could merge all the version info into a single line. Something like:

```
printf("%s version: %s (commit %s)\n", PACKAGE_NAME, PACKAGE_VERSION, GIT_COMMIT);
```